includes/Modules/Sign_In_With_Google.php

```diff
@@ -107,4 +109,68 @@ protected function setup_assets() {
 	protected function setup_settings() {
 		return new Settings( $this->options );
 	}
+
+	/**
+	 * Renders the sign in button.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function render_signin_button() {
+		$settings = $this->get_settings()->get();
+
+		$redirect_url = site_url( '/auth/google' );
+
+		if ( substr( site_url(), 0, 5 ) !== 'https' || ! $settings['clientID'] ) {
```

Comment:
We already have `$redirect_url` that we can check instead of calling `site_url` again.